### ISSUE TITLE
Cannot compile Crystal 0.30.0 on Debian

### ISSUE THREAD

**Issue by andrius** [Homebrew/linuxbrew-core#14735]
**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [ ] are reporting a bug others will be able to reproduce and not asking a question or requesting software. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh. To get software added or changed in Homebrew please file a [Pull Request](https://github.com/Homebrew/linuxbrew-core/blob/master/CONTRIBUTING.md)
- [ ] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, Homebrew/homebrew-core formula (not cask) on macOS? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new/choose. If it's a Linux problem please file this issue at https://github.com/Homebrew/linuxbrew-core/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [x] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [ ] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:

- What you were trying to do (and why)

I've tried to upgrade my linuxbrew and crystal-lang from version 0.29.0 to 0.30.0.

Ran `brew update && brew upgrade` two times, then have completely removed crystal 0.29.0 and executed `brew install crystal` to get 0.30.0 compiled on my VPS.

I can reproduce issue on another server

- What happened (include command output)

It try to compile it (top, htop, but in a while spawn multiple threads (520 in my case) with following output:

```
11598 98.9 21.7 1829568 1780184 pts/2 RNl+ 12:26   1:39 /tmp/crystal-20190810-7831-1g3hw8s/crystal-0.30.0/boot/bin/../lib/crystal/bin/crystal build -D without_openssl -D without_zlib -D preview_overflow -o .build/crystal src/compiler/crystal.cr --release --no-debug
```

- What you expected to happen

Expected to get crystal 0.30.0 installed

- Step-by-step reproduction instructions (by running `brew install` commands)

On debian 9.x  do `brew install crystal`

- Included `brew gist-logs crystal`:

Debian 9

https://gist.github.com/10155888d5555ee1d5bc70fe8203cd84

and Debian 10

https://gist.github.com/andrius/b8f3d4b64317993434152f66fa79592d

**Comment by iMichka** [comment]
We are trying to compile it too: #14710

Looks like it takes more than 6 hours to compile. I'll try on my computer. Maybe there is a bug in their build setup.

**Comment by iMichka** [comment]
Fixed by #14710
